stories/index.js

```diff
@@ -41,7 +41,47 @@ storiesOf('Picker', module)
       showPreview={boolean('Show preview', true)}
       showSkinTones={boolean('Show skin tones', true)}
       custom={CUSTOM_EMOJIS}
-    />
+    />))
+    .add('not-found-img', () => (
+      <Picker
+        onClick={action('clicked')}
+        onSelect={action('selected')}
+        onSkinChange={action('skin changed')}
+        native={boolean('Unicode', true)}
+        set={select('Emoji pack', SETS, SETS[0])}
+        emojiSize={number('Emoji size', 24)}
+        perLine={number('Per line', 9)}
+        title={text('Idle text', 'Your Title Here')}
+        emoji={text('Idle emoji', 'department_store')}
+        defaultSkin={number('Default skin tone', 1)}
+        color={color('Highlight color', '#ae65c5')}
+        showPreview={boolean('Show preview', true)}
+        showSkinTones={boolean('Show skin tones', true)}
+        custom={CUSTOM_EMOJIS}
+        notFound={
```

Comment:
I think we don't need all the props other than `notFound` here, right? 

The `default` story handles all the knobs that you can easily configure. 
These extra stories should just showcase the empty state.